apps/next/src/app/pages/single-page/single-page.scss

```
@@ -1,3 +1,9 @@
 :host {
   display: block;
 }
+
+.next-single-page-grid {
+  display: grid;
+  grid-template-columns: 0.3fr 1fr 0.3fr;
```

I do prefer whole values here, it is a ratio after all.
```suggestion
  grid-template-columns: 1fr 3fr 1fr;
```